tasks/tugboat.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+
+tasks:
+  online:tugboat:
```

Comment:
This should be called with the existing script.
* https://github.com/Lullabot/drainpipe/blob/main/.tugboat/steps/4-online.sh#L9